addons/knative/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: knative
+version: 0.1.0
+dependencies:
+  - name: knative
+    repository: file:///.
+    version: 0.1.0
```

Comment:
@fabiolrodriguez this is not needed, this should not be a dependency

Reply:
removed.